_config.yml

```diff
@@ -303,66 +363,60 @@ defaults:
       grand_parent: SQLFlow
       nav_order: 3
   - scope:
-      path: "elasticdl/doc/data_preparation_design.md"
+      path: "elasticdl/docs/designs/data_preparation.md"
```

Comment:
I propose to public the following documents for elasticdl. And the order in the website is also the same as follows:

https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/tutorials/get_started.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/overall.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/dynamic_data_sharding.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/concepts.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/parameter_server.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/high_performance_ps.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/distributed_embedding_layer.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/allreduce.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/elastic_sync_sgd.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/async_sgd.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/model_serving.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/data_preparation.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/data_io_pipeline.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/data_transform.md

Reply:
done